Formula/slither-analyzer.rb

```diff
@@ -0,0 +1,50 @@
+class SlitherAnalyzer < Formula
+  include Language::Python::Virtualenv
+
+  desc "Solidity static analysis framework written in Python 3"
+  homepage "https://github.com/crytic/slither"
+  url "https://github.com/crytic/slither/archive/refs/tags/0.8.3.tar.gz"
```

Comment:
Would be nice for these tarballs to get published on PyPI. It's a lot easier for `brew` to automatically update `resource`s that way.

Reply:
I think Slither used to have them, but I don't know what happened there. I'll ask to see if we can have them back on the next release.